app/models/manageiq/providers/workflows/automation_manager/configuration_script_source.rb

```diff
@@ -76,8 +76,26 @@ def sync_from_content(to_delete)
   end
 
   def create_workflow_from_payload(name, payload)
+    floe_workflow, payload_error =
+      begin
+        Floe::Workflow.new(payload)
```

Comment:
I had this depend on https://github.com/ManageIQ/manageiq-providers-workflows/pull/59 because we were already loading the Floe::Workflow there